manifests/create-cert.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    cat <<EOF
+Generate certificate suitable for use with an sidecar-injector webhook service.
```

Comment:
`sidecar-injector `? 

Where does this script come from? Do we should give a link if it is a modified version.